### PR TITLE
Network.to_json should handle numpy.ndarray correctly

### PR DESCRIPTION
**Pull request by yanboliang**
Pick the fix at https://github.com/keras-team/keras/pull/10754 to ```tf.keras```.

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by caisq** [comment]
Can you review this please, @fchollet?

**Review by fchollet — DISMISSED** [review]
Thanks for the PR.

**Comment by yanboliang** [comment]
It looks the pylint check failed is not relevant to this PR, I pulled and rebased master code, wish it can pass test.

**Comment by yanboliang** [comment]
I fixed the bad indentation, can anyone trigger the build again? Thanks.

**Comment by caisq** [comment]
@yanboliang Done

**Comment by yanboliang** [comment]
@caisq I fixed the bad indentation again, can you re-trigger it? Thanks. BTW, do you know how I can trigger python lint checker locally before I submit the PR?

**Comment by caisq** [comment]
@yanboliang With the necessary dependencies installed, you should be able to run it with this script (from the root directory of the repo): https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/ci_sanity.sh

**Comment by yanboliang** [comment]
@caisq Great. I just run that script and verified this PR can pass sanity check locally. Can you trigger this test again? I'm sorry to bother you again. Thanks.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by yanboliang** [comment]
Gently ping @fchollet @caisq . Would mind to have a look at this? Thanks.

**Comment by yanboliang** [comment]
@caisq Thanks!
